examples/datasource-streaming-websocket/streaming-websocket-plugin/src/ConfigEditor.tsx

```diff
@@ -24,13 +22,16 @@ export class ConfigEditor extends PureComponent<Props, State> {
     const { jsonData } = options;
 
     return (
-      <FormField
-        label="WebSocket server URL"
-        labelWidth={10}
-        inputWidth={20}
-        onChange={this.onURLChange}
-        value={jsonData.url || ''}
-      />
+      <InlineField label="URL" labelWidth={10} tooltip="Supported schemes: WebSocket (ws://) or (wss://)">
```

Comment:
Nit: Can we set the labelWidth to 14 so that it aligns with the name field above? See https://github.com/grafana/grafana/pull/86179